### ISSUE TITLE
How to integrate biotype counts generated by featureCounts into a MultiQC report

### ISSUE THREAD

**Issue by HardingZhang** [MultiQC/MultiQC#2772]
### Description of feature

Could you please tell me how to process this txt file so that MultiQC can parse it and plot the biotype counts? Also, how can I include the rRNA or protein-coding percentage in the general statistics table?

# Program:featureCounts v1.5.0; Command:"featureCounts" "-a" "genes.gtf" "-g" "gene_type" "-o" "./RL2024_biotype.featureCounts.txt" "./star/*_Aligned.sortedByCoord.out.bam"
Geneid	sample1	sample2	sample3	sample4	sample5	sample6	sample7	sample8	sample9	sample10	sample11	sample12	sample13	sample14	sample15	sample16	sample17	sample18	sample19	sample20	sample21	sample22	sample23	sample24	sample25	sample26	sample27	sample28	sample29	sample30	sample31	sample32	sample33	sample34	sample35	sample36	sample37	sample38	sample39	sample40	sample41	sample42	sample43	sample44	sample45	sample46	sample47	sample48	sample49	sample50	sample51	sample52	sample53	sample54	sample55	sample56	sample57	sample58	sample59	sample60	sample61	sample62	sample63	sample64	sample65	sample66	sample67	sample68	sample69	sample70	sample71	sample72	sample73	sample74	sample75	sample76	sample77	sample78	sample79	sample80	sample81	sample82	sample83	sample84	sample85	sample86	sample87	sample88	sample89	sample90	sample91	sample92	sample93	sample94	sample95	sample96	sample97	sample98	sample99	sample100	sample101	sample102	sample103	sample104	sample105	sample106	sample107	sample108	sample109	sample110	sample111	sample112	sample113	sample114	sample115	sample116	sample117	sample118	sample119	sample120	sample121	sample122	sample123	sample124	sample125	sample126	sample127	sample128	sample129	sample130	sample131	sample132	sample133	sample134	sample135	sample136	sample137	sample138	sample139	sample140	sample141	sample142	sample143	sample144	sample145	sample146	sample147	sample148	sample149	sample150	sample151	sample152	sample153	sample154	sample155	sample156	sample157	sample158	sample159	sample160	sample161	sample162	sample163	sample164	sample165	sample166	sample167	sample168	sample169	sample170	sample171	sample172	sample173	sample174	sample175	sample176	sample177	sample178	sample179	sample180	sample181	sample182	sample183	sample184	sample185	sample186	sample187	sample188	sample189	sample190	sample191	sample192
TEC	486	521	275	504	444	267	492	498	515	187	930	632	822	551	70	70	590	431	344	610	518	170	576	382	453	397	446	475	74	600	643	639	281	521	528	463	315	355	728	475	485	393	486	512	399	133	520	520	556	718	1036	480	235	679	385	471	1061	242	732	277	512	359	569	675	703	206	212	420	1	396	278	497	127	358	545	300	560	148	351	688	306	421	4	242	303	836	513	381	409	307	449	1586	239	229	441	552	257	445	105	900	611	179	108	52	305	23	5511	47	52	133	287	13	66	97	213	75	16	149	59	118	79	95	140	382	43	86	49	94	67	821	370	119	578	473	215	516	430	94	384	113	135	101	774	169	231	224	1443	35	33	385	132	565	469	185	260	304	133	427	310	282	717	64	164	376	13	179	321	295	132	68	159	264	1106	38	38	136	308	176	327	300	240	288	346	171	94	63	266	848	266	356	268	374
...
[RL2024_biotype.featureCounts.txt](https://github.com/user-attachments/files/16643554/RL2024_biotype.featureCounts.txt)

**Comment by vladsavelyev** [comment]
The MultiQC `featurecounts` module can only read the summary files, see docs: https://multiqc.info/modules/featurecounts/ and examples: https://github.com/MultiQC/test-data/tree/main/data/modules/featurecounts

**Comment by HardingZhang** [comment]
I'm sorry I didn't express my question clearly. The key issue I'm trying to fix is about the biotype counts. I tried process the biotype counts results to match the  format of /modules/featurecounts/multiple_files.counts.txt.summary, with the only difference being in the content of the first column. However, multiQC still couldn't recognize it. I also searched for similar issues, like https://github.com/MultiQC/MultiQC_NGI/issues/11 and https://github.com/SciLifeLab/NGI-RNAseq/pull/26, but I haven't been able to find a successful solution.

**Comment by HardingZhang** [comment]
> The MultiQC `featurecounts` module can only read the summary files, see docs: https://multiqc.info/modules/featurecounts/ and examples: https://github.com/MultiQC/test-data/tree/main/data/modules/featurecounts
I'm sorry I didn't express my question clearly. The key issue I'm trying to fix is about the biotype counts. I tried process the biotype counts results to match the format of /modules/featurecounts/multiple_files.counts.txt.summary, with the only difference being in the content of the first column. However, multiQC still couldn't recognize it. I also searched for similar issues, like MultiQC/MultiQC_NGI#11 and SciLifeLab/NGI-RNAseq#26, but I haven't been able to find a successful solution.
